packages/helpermodules/update_config.py

```diff
@@ -1491,3 +1491,26 @@ def upgrade(topic: str, payload) -> None:
 
         self._loop_all_received_topics(upgrade)
         Pub().pub("openWB/system/datastore_version", 44)
+
+    def upgrade_datastore_44(self) -> None:
+        def upgrade(topic: str, payload) -> Optional[dict]:
+            if re.search("openWB/vehicle/template/charge_template/[0-9]+$", topic) is not None:
+                payload = decode_payload(payload)
+                if "disable_after_unplug" in payload:
+                    updated_payload = payload
+                    updated_payload["disable_after_unplug"] = {}
+                    payload.pop("disable_after_unplug")
+                    return {topic: updated_payload}
+            if re.search("openWB/chargepoint/template/[0-9]+$", topic) is not None:
+                payload = decode_payload(payload)
+                if "rfid_enabling" in payload:
+                    updated_payload = payload
+                    updated_payload["rfid_enabling"] = {}
+                    payload.pop("rfid_enabling")
+                    return {topic: updated_payload}
+                if "disable_after_unplug" not in payload:
+                    updated_payload = payload
+                    updated_payload.update({"disable_after_unplug": False})
+                    return {topic: updated_payload}
+        self._loop_all_received_topics(upgrade)
+        self.__update_topic("openWB/system/datastore_version", 45)
```

Comment:
```suggestion

    def upgrade_datastore_44(self) -> None:
        def upgrade(topic: str, payload) -> Optional[dict]:
            if re.search("openWB/vehicle/template/charge_template/[0-9]+$", topic) is not None:
                payload = decode_payload(payload)
                if "disable_after_unplug" in payload:
                    updated_payload = payload
                    disable_after_unplug  = updated_payload["disable_after_unplug"]
                    payload.pop("disable_after_unplug")
                    return {topic: updated_payload}
            if re.search("openWB/chargepoint/template/[0-9]+$", topic) is not None:
                payload = decode_payload(payload)
                if "rfid_enabling" in payload:
                    updated_payload = payload
                    payload.pop("rfid_enabling")
                    return {topic: updated_payload}
                if "disable_after_unplug" not in payload:
                    updated_payload = payload
                    updated_payload.update({"disable_after_unplug": disable_after_unplug})
                    return {topic: updated_payload}
        self._loop_all_received_topics(upgrade)
        self.__update_topic("openWB/system/datastore_version", 45)
```

Reply:
l.1501 macht das Sinn?

Reply:
Getestet funktioniert die Datastore Funktion, die ich im PR habe,...was ist der Unterschied?

Reply:
Das, was der Kunde in der alten Einstellung eingestellt hatte, soll mit übernommen werden. Wenn der Kunde "Sperren nach Abstecken" aktiv hatte und ein Update macht, steht die Einstellung auf einmal auf False und sein Ladepunkt ist nicht mehr gesperrt.

Reply:
Achsooo. l.1501 merkt sich den Zustand und in l.1512 wird der übernommen. Danke